YAMLSerialization.h

```diff
@@ -10,27 +10,27 @@
 #import <Foundation/Foundation.h>
 
 // Mimics NSPropertyListMutabilityOptions
-typedef enum {
-    kYAMLReadOptionImmutable                  = 0x0000000000000001,
-    kYAMLReadOptionMutableContainers          = 0x0000000000000010,
-    kYAMLReadOptionMutableContainersAndLeaves = 0x0000000000000110,
-    kYAMLReadOptionStringScalars              = 0x0000000000001000
-} YAMLReadOptions;
-
-typedef enum {
+typedef NS_OPTIONS(NSUInteger, YAMLReadOptions) {
+    kYAMLReadOptionImmutable                  = 1 << 0,
+    kYAMLReadOptionMutableContainers          = 1 << 1,
+    kYAMLReadOptionMutableContainersAndLeaves = 1 << 2,
```

Comment:
Oops, sorry. I overlooked that. Thanks for pointing out. I've pushed new commit.